jerry-core/debugger/debugger.c

```diff
@@ -202,6 +215,24 @@ jerry_debugger_send_eval (const lit_utf8_byte_t *eval_string_p, /**< evaluated s
 } /* jerry_debugger_send_eval */
 
 /**
+ * Suspend execution for a given time.
+ * Note: If the platform does not have nanosleep or usleep, this function does not sleep at all.
+ */
+static void
+jerry_debugger_sleep (int milliseconds) /**< suspending time */
+{
+#ifdef HAVE_TIME_H
+  nanosleep (&(const struct timespec)
+  {
+    milliseconds / 1000, (milliseconds % 1000) * 1000000L /* Seconds, nanoseconds */
+  }
+  , NULL);
+#elif defined (HAVE_UNISTD_H)
+  usleep ((__useconds_t) milliseconds * 1000);
```

Comment:
I'm not completely sure about this but it's suspicious: are you sure that `__useconds_t` is the type that should be used here? double underscores usually denote internal things. according to manuals, the declaration of usleep is `int usleep(useconds_t usec);`, i.e., the public type has no underscore prefix. please check